documentation/src/main/asciidoc/topics/proc_displaying-metrics-st-operation.adoc

```diff
@@ -0,0 +1,41 @@
+[id='displaying-metrics-st-operationadoc_{context}']
+= Displaying metrics during a state transfer operation
+
+You can export time metrics for clustered caches that {brandname} redistributes across nodes. 
+
+A state transfer operation occurs when a clustered cache topology changes, such as a node joining a cluster, so the cache's data must be redistributed to all nodes in a cluster. 
```

Comment:
As I mentioned in the Gdocs, I think we should replace/remove the `all nodes`.  

And some nitpicking:
We reverse these two sentences and put them together. We start saying what is a state transfer and that we expose metrics for it.

Reply:
Updated to:

> A state transfer operation occurs when a clustered cache topology changes, such as a node joining or leaving a cluster.
> During a state transfer operation, {brandname} exports metrics from each cache, so that you can determine the status of a cache entry.

WDYT?

Reply:
Sounds good.